test/box/gh-6092-invalid-listen-uri.result

```diff
@@ -20,7 +20,7 @@ old_listen = box.cfg.listen
 ...
 box.cfg({ listen = bad_uri })
 ---
-- error: can't resolve uri for bind, called on fd -1
+- error: 'can''t resolve uri for bind, called on fd -1: Resource temporarily unavailable'
```

Comment:
Some errors have system-dependent error messages. For example, this one won't pass on Mac:
```
[001] box/gh-6092-invalid-listen-uri.test.lua                         [ fail ]
[001] 
[001] Test failed! Result content mismatch:
[001] --- box/gh-6092-invalid-listen-uri.result	Sat Nov 27 00:51:41 2021
[001] +++ var/rejects/box/gh-6092-invalid-listen-uri.reject	Sat Nov 27 00:58:46 2021
[001] @@ -20,7 +20,7 @@
[001]  ...
[001]  box.cfg({ listen = bad_uri })
[001]  ---
[001] -- error: 'can''t resolve uri for bind, called on fd -1: Resource temporarily unavailable'
[001] +- error: 'can''t resolve uri for bind, called on fd -1: Undefined error: 0'
[001]  ...
[001]  conn = netbox.connect(old_listen)
```
The others worked fine. So here it might be good to use `:match()` on the message to check for something platform-agnostic. Such as `err.msg:match("can't resolve uri")`.

Reply:
Thanks for catching this! CI is broken on OS X so I wouldn't find out about this issue otherwise. Fixed as follows:

<details>
<summary>diff</summary>

```diff
diff --git a/test/box/gh-6092-invalid-listen-uri.result b/test/box/gh-6092-invalid-listen-uri.result
index f204ae622c8b..5b65741806f9 100644
--- a/test/box/gh-6092-invalid-listen-uri.result
+++ b/test/box/gh-6092-invalid-listen-uri.result
@@ -18,9 +18,16 @@ bad_uri = "baduribaduri:1"
 old_listen = box.cfg.listen
 ---
 ...
-box.cfg({ listen = bad_uri })
+success, err = pcall(box.cfg, {listen = bad_uri})
 ---
-- error: 'can''t resolve uri for bind, called on fd -1: Resource temporarily unavailable'
+...
+assert(not success)
+---
+- true
+...
+assert(err.message:match("can't resolve uri for bind") ~= nil)
+---
+- true
 ...
 conn = netbox.connect(old_listen)
 ---
```

</details>